src/main/java/com/untamedears/realisticbiomes/utils/RealisticBiomesGUI.java

```diff
@@ -96,6 +97,31 @@ public void showRBOverview() {
 				}
 			}
 			lore.add(ChatColor.DARK_AQUA + "Biome Multiplier: " + ChatColor.GRAY + biomeMultiplier);
+			if (plant.getAllowBoneMeal() || !plant.getNeedsLight()
+					|| plant.getSoilBoniPerLevel().size() != 0 || plant.getMaximumSoilLayers() != 0
```

Comment:
Always use .isEmpty() instead of comparing size. It's faster for some collection implementations.